pyomo/meta.yaml

```diff
@@ -7,7 +7,8 @@ package:
 
 source:
   fn: Pyomo-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz
+  https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
```

Comment:
Got it.


On Wed, Apr 27, 2016 at 2:26 AM, Juan Luis Cano Rodríguez <
notifications@github.com> wrote:

> In pyomo/meta.yaml
> <https://github.com/CAChemE/pyomo-recipes/pull/13#discussion_r61209667>:
>
> > @@ -7,7 +7,8 @@ package:
> >
> >  source:
> >    fn: Pyomo-{{ version }}.tar.gz
> > -  url: https://pypi.python.org/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz
> > +  https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
>
> Oops, you missed url: here!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/CAChemE/pyomo-recipes/pull/13/files/055b0fae10b62fadeb9bab87503bd26d9c1289da..7928db46b6fff41772aa2192015f48839323ab4a#r61209667>
>